// Function call with wrong number of argument
package main

func a(a int) {}

func main() {
    a();
}
